backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteForHiveSuite.scala

```diff
@@ -176,4 +176,13 @@ class VeloxParquetWriteForHiveSuite extends GlutenQueryTest with SQLTestUtils {
       checkAnswer(sql("SELECT * FROM t"), Row(1))
     }
   }
+
+  test("native writer support CreateHiveTableAsSelectCommand") {
+    withTable("t", "t2") {
```

Comment:
`t2` is not used in the test, it should be removed.
BTW, `create table stored as orc` should also be needed in this test.

Reply:
removed, this `VeloxParquetWriteForHiveSuite` suite is specifically for Parquet, and the code paths for ORC and Parquet are consistent.